### DISCUSSION TITLE
How to center text of a table column

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#6918] [Help] [answered]
Hi,
How can I center the contents of a table column.? I have the following code

`        Tables\Columns\TextColumn::make('documentCount')
          ->label(__('Documentos'))
          ->extraAttributes(['class' => 'text-center'])
          ->formatStateUsing(fn ($record) => $record->transferDocuments()->count()),`

but it is not aligning  to the center. The row is clickable so the column content is embedded inside a button.
The extra attributes do get added but not on the button which is wrapping the contents.

Thanks

**Comment by RX-77IT** [comment, marked as answer]
Can you try to swap `->extraAttributes(['class' => 'text-center'])` with `->alignCenter()` ?

I found it here https://github.com/filamentphp/filament/blob/633ea6f739ced12b4dd4d9403ce5bc66efa1e74e/packages/tables/src/Columns/Concerns/HasAlignment.php

**Reply by jwhoami** [reply]
Perfect. Tks man. Did not know about these options.
